terraform/services/rds-api/terraform.tf

```diff
@@ -0,0 +1,9 @@
+provider "aws" {
+  region = "us-east-1"
```

Comment:
The aws provider definition should look more like https://github.com/CMSgov/ab2d-bcda-dpc-platform/blob/main/terraform/services/api-waf/terraform.tf#L2-L11